examples/polkadot.ts

```diff
@@ -100,7 +102,13 @@ async function main(): Promise<void> {
   );
 
   // Sign a payload. This operation should be performed on an offline device.
-  const signature = signWith(registry, alice, signingPayload);
+  // Important! The registry needs to be passed into the signing function. It
+  // needs to be updated with latest metadata, so before calling this function,
+  // make sure to run `registry.setMetadata(metadata)` first.
```

Comment:
This isn't true if you pass the metadata into the signing function, so should be run `setMetadata` first and leave `signWith` unchanged, or keep the `signWith` changes and pass in `metadataRpc`.